### PR TITLE
Mccalluc/get chuck running again

### LINKED ISSUE #2684: Dev start-up fails: "Node Sass does not yet support your current environment"
Rebooted earlier this morning, just now started a new feature branch from main, and `dev-start.sh` produced this error message:
```
ERROR in ./app/static/js/components/searchPage/Search.scss (./node_modules/css-loader/dist/cjs.js!./node_modules/sass-loader/dist/cjs.js!./app/static/js/components/searchPage/Search.scss)
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (93)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v4.14.1
    at module.exports (/Users/chuck/github/hubmap/portal-ui/context/node_modules/node-sass/lib/binding.js:13:13)
    at Object.<anonymous> (/Users/chuck/github/hubmap/portal-ui/context/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at getDefaultSassImplementation (/Users/chuck/github/hubmap/portal-ui/context/node_modules/sass-loader/dist/getDefaultSassImplementation.js:24:10)
    at getSassImplementation (/Users/chuck/github/hubmap/portal-ui/context/node_modules/sass-loader/dist/getSassImplementation.js:19:72)
```

We only specify the lower bound on npm dependencies... possible that there has been some breaking change?

I'll `npm ci` and try again.

### LINKED ISSUE #2695: in dev-start.sh, check version numbers
Read the configuration (either from docker or CI), and confirm that the python and node versions in the local environment match what is expected.

Not sure if there's an elegant solution for keeping these all in sync that doesn't add complexity.

### PR DESCRIPTION
**Pull request by mccalluc**
- fix #2695
- fix #2684

### CODE REVIEW

**Comment by mccalluc** [comment]
Storybook error:
```
info @storybook/react v6.3.7
info
ERR! /Users/chuck/github/hubmap/portal-ui/context/.storybook/main.js:16
ERR!       ...config.resolve?.alias,
ERR!                         ^
ERR!
ERR! SyntaxError: Unexpected token '.'
```

**Comment by mccalluc** [comment]
Storybook still failing locally for me... but it also fails when I checkout main, so this at least isn't making things worse. Probably file a separate issue for that?

**Comment by john-conroy** [comment]
For the storybook error maybe it's an issue with the optional chaining operator?  https://node.green/#ES2020-features-optional-chaining-operator----- indicates support starts with 14.

Let's pin it to a stable 14.x.x version? 12.16.3 was chosen somewhat arbitrarily and I've been using 14 locally.

**Comment by john-conroy** [comment]
We should also tackle upgrading node at some point soon as end-of-life is encroaching for 14.

**Comment by john-conroy** [comment]
We should also be sure to use the same version for docker in `context/dockerfile`.

**Comment by mccalluc** [comment]
@john-conroy : Thanks: I'll upgrade node here. I've branched another PR from this that keeps Docker in sync: #2703

**Comment by mccalluc** [comment]
Upgraded... but haven't confirmed locally that everything runs. Perhaps look at #2703 at first, and if it's good, I'm merge that into this, and then confirm that docker and dev-start work with the node upgrade. Your call.

**Comment by mccalluc** [comment]
Update: dev-start works, but docker doesn't... probably why cypress CI fails. I'll switch this back to draft for now...

**Comment by mccalluc** [comment]
and docker.sh now works locally.

File: README.md [hunk 1]
```diff
@@ -85,8 +85,8 @@ All designs are in [Figma](https://www.figma.com/files/team/834568130405102661/H
 
 ### Prerequisites
 - `git`: Suggest [installing Apple XCode](https://developer.apple.com/xcode/).
-- `python 3.9`: Suggest [installing miniconda](https://docs.conda.io/en/latest/miniconda.html#macosx-installers) and [creating a new conda environment](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-with-commands) with Python 3.9.
-- `nodejs/npm`: Suggest [installing nodejs](https://nodejs.org/en/).
+- `python 3.9`: Suggest [installing miniconda](https://docs.conda.io/en/latest/miniconda.html#macosx-installers) and [creating a new conda environment](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-with-commands): `conda create -n portal python=$(cat .python-version)`
+- `nodejs/npm`: Suggest [installing nvm](https://github.com/nvm-sh/nvm#installing-and-updating) and then using it it install the appropriate node version: `nvm install`.
```
Comment (john-conroy) [inline README.md:89]:
```suggestion
- `nodejs/npm`: Suggest [installing nvm](https://github.com/nvm-sh/nvm#installing-and-updating) and then using it to install the appropriate node version: `nvm install`.
```

**Review by john-conroy — APPROVED** [review]

**Pull request merged**